tests/Integration/RedirectsTest.php

```diff
@@ -9,6 +9,18 @@ final class RedirectsTest extends TestCase {
 
 	public function get_redirect_data() {
 		return array(
+			'redirect_unicode_in_path'  => array(
+				// https://www.w3.org/International/articles/idn-and-iri/
+				'/JP納豆',
+				'http://example.com',
+			),
+
+			'redirect_unicode_in_path'  => array(
```

Comment:
Update this so that it is a different key to the one above.
```suggestion
			'redirect Arabic in path'  => array(
```